### PR TITLE
Control pt-web-vnc-desktop in VNC action page

### PR DESCRIPTION
**Pull request by jcapona**
| Status  | Ticket/Issue |
| :---: | :--: |
| Ready/Hold | [Ticket](https://pi-top.atlassian.net/browse/OS-1332) |

#### Main changes
- VNC toggle in the Settings Page also controls the `pt-web-vnc-desktop` systemd service.

#### Screenshots (feature, test output, profiling, dev tools etc)

[insert screenshots here]

#### Other notes (e.g. implementation quirks, edge cases, questions / issues)
-

#### Manual testing tips
-

#### Tag anyone who definitely needs to review or help
-

### CODE REVIEW

File: debian/control [hunk 1]
```diff
@@ -18,6 +18,8 @@ Depends:
 Recommends:
  autorandr (>= 1.9),
  further-link,
+# Control desktop sharing via browser
+ pt-web-vnc,
```
Comment (angusjfw) [inline debian/control:22]: if we can version these dependencies we should 
```suggestion
 pt-web-vnc (>= 0.0.2),
```

**Review by angusjfw — COMMENTED** [review]
this works, but is dependent on unmerged pt-web-vnc-desktop

**Review by angusjfw — COMMENTED** [review]
LGTM, pt-web-vnc is released now, but the lgtm bot seems to think get_systemd_enabled_state shouldn't take arguments - could you try to work out what it's on about

**Comment by jcapona** [comment]
> LGTM, pt-web-vnc is released now, but the lgtm bot seems to think get_systemd_enabled_state shouldn't take arguments - could you try to work out what it's on about

The mocked `get_systemd_enabled_state` function had the incorrect number of arguments... just fixed it

**Review by angusjfw — APPROVED** [review]
